latex/generator-tex.sh

```
@@ -0,0 +1,216 @@
+
+    # Count dashes to have according to percentages
+    dashesToHavePerColumn="$(echo "$dash_pct_json" | jq 'map((. * 100)|ceil)')"
+
+    echo "  Dashes per column: $dash_json (sum=$dash_sum) Percentages: $dash_pct_json, need more dashes to reach at 80 chars, distributing as: $dashesToHavePerColumn"
+
+    # Build new table separator line from dashesToHavePerColumn
+    new_sep_line="|"
+    while read -r count; do
+      # enforce minimum 3 dashes per Markdown spec
+      [ "$count" -lt 3 ] && count=3
+      new_sep_line="${new_sep_line} $(printf '%*s' "$count" | tr ' ' -) |"
```

Shellcheck (0.11.0)

[warning] This format string has 2 variables, but is passed 1 argument.

(SC2183)